audio/src/main/java/com/voxeet/audio2/devices/MediaDevice.java

```diff
@@ -56,7 +56,7 @@ protected MediaDevice(@NonNull IMediaDeviceConnectionState mediaDeviceConnection
                           @Nullable TYPE holder,
                           @Nullable String name) {
         connectionState = ConnectionState.DISCONNECTED;
-        lastConnectionStateType = LastConnectionStateType.SYSTEM;
+        lastConnectionStateType = LastConnectionStateType.PROGRAMMATIC;
```

Comment:
the default should be system because programmatic means that it was the developer who interacted with it. However if you have any counter argument, I'm open to it :p

Reply:
Yep, I have a reason for that. It cannot be SYSTEM because it will not be possible to distinguish when device gets disconnected by SYSTEM (pressing Hangup button) and when device was platform connected for the firsts time (BluetoothDevice was created). In both cases we will have connection state DISCONNECTED, connection type SYSTEM and platform connection CONNECTED.
Looking from device selection policy point of view, we connect to BT when nothing is connected and BT connection type is PROGRAMMATIC, but we don't connect to BT when connection type is SYSTEM/